conduit/plugins/importers/algod/algod_importer.go

```diff
@@ -134,6 +249,8 @@ func (algodImp *algodImporter) Init(ctx context.Context, cfg plugins.PluginConfi
 		return nil, fmt.Errorf("unable to fetch genesis file from API at %s", algodImp.cfg.NetAddr)
 	}
 
+	err = algodImp.catchupNode(initProvider)
```

Comment:
when follower node's round equals  initProvider.NextDBRound(), no catchup is needed, right?

Reply:
Right, but it's not harmful to set the sync round and call `wait-for-block-after` in this case since they should instantly succeed and also ensure the node is in a good state.